README.md

```diff
@@ -1,110 +1,37 @@
 [![Build Status](https://circleci.com/gh/mapbox/mapbox-gl-js.svg?style=svg)](https://circleci.com/gh/mapbox/mapbox-gl-js) [![Coverage Status](https://coveralls.io/repos/github/mapbox/mapbox-gl-js/badge.svg?branch=master)](https://coveralls.io/github/mapbox/mapbox-gl-js?branch=master)
 
-# Mapbox GL JS
+[Mapbox provides is an ecosystem of libraries for rendering maps on the web, iOS, Android, Qt, macOS and more.](https://www.mapbox.com/maps/). This repository holds the files and issues shared among all platforms.
```

Comment:
The Mapbox Unity SDK renders maps, but it likely will never be part of this repository. I’m sure there are other libraries that don’t fall neatly within this statement. So I think it’s still necessary to keep the “Mapbox GL” moniker around as shorthand for the map rendering libraries that target WebGL/OpenGL directly.